cli/src/electron/add.ts

```diff
@@ -1,11 +1,18 @@
 import { exec } from 'child_process';
 import { Config } from '../config';
 import { copyTemplate, hasYarn, installDeps, runTask } from '../common';
+import { readFileSync, writeFileSync } from '../util/fs';
```

Comment:
readFileSync can be removed as require is used for getting the json files

```suggestion
import { writeFileSync } from '../util/fs';
```

Reply:
Always forget about requiring JSON files